microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,162 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+<!-- ====================================================================== -->
+## 1.0.1462.37
+
+Release Date: December 12, 2022
+
+[NuGet package for WebView2 SDK 1.0.1462.37](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1462.37)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 108.0.1462.37 or higher.
+
+### General
+
+This WebView2 SDK release has the same [bug fixes](#bug-fixes-1466-prerelease) that are in WebView2 SDK 1.0.1466-prerelease.
+
+
+
+
+<!-- ====================================================================== -->
+## 1.0.1549-prerelease
+
+Release Date: December 12, 2022
+
+[NuGet package for WebView2 SDK 1.0.1549-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1549-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 110.0.1549.0 or higher.
+
+### General
+
+#### Experimental features 
+
+*  Added support for the Locale Region API:

```

Comment:
It says Locale Region but the links are for the Crash Reporting ones.

Reply:
fixed by Suggestion comments such as https://github.com/MicrosoftDocs/edge-developer/pull/2359/files#r1047928103